jobs/BUILD

```diff
@@ -5,6 +5,7 @@ py_binary(
     srcs = ["config_sort.py"],
     data = [
         ":jobs",
+        "//boskos:resources.json",
```

Comment:
it probably currently works because the `resources.json` is included in `pkg-srcs`, and so is already known to bazel.

having a filegroup-per-purpose is ideal (which is what we did with prow here), but I guess it's not essential.